server/common/mail/index.js

```diff
@@ -23,12 +25,14 @@ module.exports = {
 
 function invite(user, { origin }) {
   const href = resetUrl(origin, user);
-  const { hostname } = new URL(href);
-  const recipient = user.email;
-  const message = `
-    An account has been created for you on ${hostname}.
-    Please click <a href="${href}">here</a> to complete your registration.`;
+  const { protocol, hostname } = new URL(href);
```

Comment:
How about extracting `origin` and `hostname` instead because `origin` equals `protocol` plus `hostname` 💡